server/modules/email/success-offboarding-email.html

```diff
@@ -0,0 +1,18 @@
+<p><b>Developer Roll Off Notification</b></p>
+
+<p>
+  This is to notify you that the ${developerName}, ${developerEmail}
+  ${developerLocation}, has been rolled off from ${partnerName},
+  ${partnerLocation}.
+</p>
+
+<p>Their expected roll off date is ${rollOffDate}</p>
+
+<p>
+  Please do provide them with any support they need to ensure smooth transition
+  from their Partner Engagement.
+</p>
+
+<p>Regards,</p>
+<p>Success</p>
+
```

Comment:
Please also keep all the email templates in a folder(probably called `emailTemplates`)

Reply:
This should then be a chore on its own.

Reply:
Or I can do an extra commit for this, just to maintain sanity in our git history

Reply:
Awesome!